sidebarsGuides.js

```diff
@@ -0,0 +1,292 @@
+                                    label: 'Overview',
+                                    type: 'doc',
+                                },
+                                'app-runtime/advanced/offline/CacheableSections',
+                                'app-runtime/advanced/offline/useDhis2ConnectionStatus',
+                                'app-runtime/advanced/offline/useOnlineStatus',
+                            ],
+                        },
+                        'app-runtime/advanced/CustomDataProvider',
+                        'app-runtime/advanced/services',
+                        'app-runtime/advanced/DataEngine',
+                        'app-runtime/advanced/DataEngineLinks',
+                        'app-runtime/advanced/redux',
+                    ],
+                },
+                {
+                    href: 'https://play.dhis2.org/demo/api/apps/query-playground/index.html',
+                    label: 'Query Playground',
+                    type: 'link',
+                },
+                {
+                    id: 'app-runtime/changelog',
+                    label: 'Changelog',
+                    type: 'doc',
+                },
+            ],
+        },
+
+        {
+            label: 'Capture Plugins',
+            type: 'category',
+            collapsed: true,
+            collapsible: true,
+            link: {
+                type: 'doc',
+                id: 'capture-plugins/developer/getting-started',
+            },
+            items: [
+                'capture-plugins/developer/getting-started',
+                'capture-plugins/developer/develop-a-capture-plugin',
+                'capture-plugins/developer/configure-a-capture-plugin',
+                {
+                    'Form Field Plugin': [
+                        'capture-plugins/developer/form-field-plugins/introduction',
+                        'capture-plugins/developer/form-field-plugins/developer-details',
+                        'capture-plugins/developer/form-field-plugins/manual-setup',
+                    ],
+                    'Enrollment Plugin': [
+                        'capture-plugins/developer/enrollment-plugins/introduction',
+                        'capture-plugins/developer/enrollment-plugins/developer-details',
+                        'capture-plugins/developer/enrollment-plugins/manual-setup',
+                    ],
+                },
+            ],
+        },
+        {
+            Guides: [
+                'guides',
+                {
+                    'Local Development': [
+                        'guides/spin-up-local-instance',
+                        'guides/code-style',
+                        'guides/debug-instance',
+                    ],
+                    'Web UI Library': [
+                        'guides/ui-table',
+                        {
+                            label: 'Web UI Component Recipes',
```

Comment:
Let's move this up to under "guides", no need to nest further

Reply:
Yes, good call! This means the ui-table doc will actually move toe the UI library recipes directory